README.md

```diff
@@ -3,91 +3,83 @@
+```
 
-1. navigate to `ProbeParticleModel/examples/PTCDA_Hartree`
-2. run the example `./run.sh`
+Additionally an OpenCL Installable Client Driver (ICD) for your compute device is required:
+* Nvidia GPU: comes with the standard Nvidia driver (nvidia-driver-xxx)
+* AMD GPU: `sudo apt install mesa-opencl-icd`
+* Intel HD Graphics: `sudo apt install intel-opencl-icd`
+* CPU: `sudo apt install pocl-opencl-icd`
 
-*NOTE:* Notice that the script `run.sh` downloads and unpack LOCPOT file:
-`wget --no-check-certificate "https://www.dropbox.com/s/18eg89l89npll8x/LOCPOT.xsf.zip"`
-`unzip LOCPOT.xsf.zip`
-this is a large 3D volumetric file which contains Hartree electrostatic potential (in this example computed by VASP) which have to be provided.
+### Use ppafm Docker container
 
-##### Example 3: Pyridine with Density-overlap
+We propose to use [Docker](https://docs.docker.com/get-docker/) to make the code platform-independent.
 
-1. navigate to `ProbeParticleModel/examples/pyridineDensOverlap`
-2. run the example `./run.sh`
+Here are the steps to build and run the ppafm Docker container:
 
-*NOTE:* Notice that the script `run.sh` downloads and unpacks files `CHGCAR.xsf` & `LOCPOT.xsf` and places them in subdirectories `sample` and `tip`. These are electron density and Hartree potential which need to be provided from DFT calculation (this time from VASP).
+1. Build the image.
 
-#### Install & run GPU GUI
+```bash
+$ docker build -t ppafm:latest .
+```
+2. Execute the container.
 
-Install prerequisites (Ubuntu):
-```sh
-sudo apt install git python3-pip python3-pyqt5
-pip install matplotlib numpy pyopencl reikna ase
+```bash
+$ docker run --rm -it -v ${PWD}:/exec ppafm:latest <ppafm command>
 ```
 
-Additionally an OpenCL Installable Client Driver (ICD) for your compute device is required:
-* Nvidia GPU: comes with the standard Nvidia driver (nvidia-driver-xxx)
-* AMD GPU: `sudo apt install mesa-opencl-icd`
-* Intel HD Graphics: `sudo apt install intel-opencl-icd`
-* CPU: `sudo apt install pocl-opencl-icd`
 
-Run the GUI application:
-```sh
-./GUI/ppm-gui
-```
+## Usage examples
+
+We provide a set of examples in the `examples` directory.
+To run them, navigate to the directory and run the `run.sh` script.
+For example:
 
-In order to make the GUI application appear in the system application menu and 'Open with' context menus, link the `ppm-gui` application to a location that is on PATH, e.g. `~/.local/bin`, and install the provided .desktop file. This can be achived by running the following in the repository root:
 ```bash
-ln -s `realpath ./GUI/ppm-gui` $HOME/.local/bin
-cp ./GUI/resources/ppm-gui.desktop $HOME/.local/share/applications
+$ cd examples/PTCDA_single
+$ ./run.sh
 ```
 
-###### Usage:
+You can study the script to see how to run the simulation.
+Also, have a look at the `params.ini` file to see how to set up the simulation parameters.
+
+
+Once the simulation is finished, a number of files and folders will be created.
+
+### GUI usage
```

Comment:
I would move this whole section to the wiki as well. The GUI needs more documentation anyway.